packages/core-mobile/e2e/pages/manageTokens.page.ts

```diff
@@ -37,6 +38,36 @@ class ManageTokensPage {
   async inputCustomToken(address: string) {
     await Action.setInputText(this.inputContractAddress, address, 0)
   }
+
+  async hideToken(token: string) {
+    await commonElsPage.typeSearchBar(token)
+    // TryCatch Phrase is for test requirment
+    try {
+      await Action.waitForElement(by.id(`${token}_blocked`))
+      await Action.tap(by.id(`${token}_blocked`))
+      console.log("Display the token if it's already hidden")
+    } catch (e) {
+      console.log("It's already displayed on token list")
+    }
+    // Hide the token
+    await Action.tap(by.id(`${token}_displayed`))
+    await commonElsPage.goBack()
+  }
+
+  async showToken(token: string) {
+    await commonElsPage.typeSearchBar(token)
+    // TryCatch Phrase is for test requirment
+    try {
+      await Action.waitForElement(by.id(`${token}_display`))
+      await Action.tap(by.id(`${token}_blocked`))
+      console.log("Block the token if it's already displayed")
+    } catch (e) {
+      console.log("It's already blocked")
+    }
+    // Display the token
+    await Action.tap(by.id(`${token}_blocked`))
+    await commonElsPage.goBack()
+  }
```

Comment:
These methods are almost the same and should be combined into one.  Something like this?
```
async showToken(token: string, tokenStatus: "blocked") {
    await commonElsPage.typeSearchBar(token)
    // TryCatch Phrase is for test requirement
    try {
      await Action.waitForElement(by.id(`${token}_${tokenStatus}`))
      await Action.tap(by.id(`${token}_${tokenStatus}`))
      console.log("Block the token if it's already displayed")
    } catch (e) {
      console.log("It's already blocked")
    }
    // Display the token
    await Action.tap(by.id(`${token}_${tokenStatus}`))
    await commonElsPage.goBack()
  }